src/main/java/com/hivemq/extensions/kafka/api/transformers/kafkatomqtt/KafkaToMqttInitInput.java

```diff
@@ -21,7 +21,7 @@
 import com.hivemq.extensions.kafka.api.transformers.TransformerInitInput;
 
 /**
- * Provides context for the set up of a {@link KafkaToMqttTransformer}.
+ * Provides context for the set-up of a {@link KafkaToMqttTransformer}.
```

Comment:
My understanding is that you should use `setup` on this context https://www.merriam-webster.com/dictionary/setup